File: _scratch/run/detect_capital/sol.cpp
#include <bits/stdc++.h>
using namespace std;

class Solution {
public:
    bool detectCapitalUse(string s) {
        auto isUpper = [](const string& x) { for (char c : x) if (!isupper((unsigned char)c)) return false; return true; };
        auto isLower = [](const string& x) { for (char c : x) if (!islower((unsigned char)c)) return false; return true; };
        if (isUpper(s) || isLower(s)) return true;
        return isupper((unsigned char)s[0]) && isLower(s.substr(1));
    }
};
int main(){string l;getline(cin,l);int T=stoi(l);string s;for(int i=0;i<T;i++){if(!getline(cin,s))s="";cout<<(Solution().detectCapitalUse(s)?"true":"false")<<"\n";}}

File: _scratch/run/detect_capital/sol.py
class Solution:
    def detectCapitalUse(self, s):
        return s.isupper() or s.islower() or (s[0].isupper() and s[1:].islower())
import sys
_l=sys.stdin.read().split('\n')
_T=int(_l[0]);_o=[]
for _i in range(_T):
    _s=_l[1+_i] if 1+_i<len(_l) else ""
    _o.append('true' if Solution().detectCapitalUse(_s) else 'false')
print('\n'.join(_o))